assets/images/avatar/oppia_black_100px.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Let's rename the file to oppia_avatar_100px.svg or similar, since the replaced avatar may not be black.